resources/views/emails/no_paper.blade.php

```diff
@@ -0,0 +1,6 @@
+@component('mail::message')
```

Comment:
Please use `@lang` directives here. You can use the other mails as examples. I will add the translations when it's done.